bin/pygrb/pycbc_make_offline_grb_workflow

```diff
@@ -418,43 +448,52 @@ ppDir = os.path.join(currDir, "post_processing")
 os.makedirs(ppDir)
 post_proc_method = wflow.cp.get_opt_tags("workflow-postproc",
                                          "postproc-method", tags)
-
+pp_files = _workflow.FileList([])
+results_files = _workflow.FileList([])
 if post_proc_method == "PYGRB_OFFLINE":
     pp_files = _workflow.setup_pygrb_pp_workflow(wflow, ppDir, segDir,
                                       sciSegs[ifo][0], inspiral_files,
                                       injs, inj_insp_files, inj_tags)
-
-# TODO: Remove all coh_ptf post-processing
-if post_proc_method in ["COH_PTF_WORKFLOW", "COH_PTF_OFFLINE",
-                        "COH_PTF_ONLINE"]:
-    from pylal import pygrb_cohptf_pp
-    # Add parsed config file so it can be linked from summary page
-    cp_file_name = workflow_name + ".ini"
-    cp_file_url = "file://localhost%s/%s" % (runDir, cp_file_name)
-    cp_file = _workflow.File(ifos, cp_file_name, sciSegs[ifo][0],
-                             file_url=cp_file_url)
-    cp_file.PFN(cp_file.cache_entry.path, site="local")
-
-    # Generate post-processing workflow
-    html_dir = wflow.cp.get("workflow", "html-dir")
-    pp_files = pygrb_cohptf_pp.setup_coh_PTF_post_processing(wflow,
-            inspiral_files, inspiral_cache, ppDir, segDir,
-            injection_trigger_files=inj_insp_files, injection_files=injs,
-            injection_trigger_caches=inj_insp_caches,
-            timeslide_trigger_files=inspiral_ts_files,
-            injection_caches=inj_caches, config_file=cp_file, web_dir=html_dir,
-            segments_plot=segs_plot, ifos=ifos, inj_tags=inj_tags)
-
-    # Retrieve style files for webpage
-    summary_files = _workflow.get_coh_PTF_files(wflow.cp, ifos, ppDir,
-                                                summary_files=True)
-
-    pp_files.extend(_workflow.FileList([cp_file]))
-    pp_files.extend(summary_files)
+    # pp_files[1] ALLTIMES_CLUSTERED File
+    # pp_files[3] ONSOURCE_CLUSTERED File
+    # pp_files[-2] FOUNDMISSED FileList
+    # pp_files[-1] FOUNDMISSED-FILTERED FileList
```

Comment:
This comment makes me wonder if different data structure may be beneficial here in terms of code clarity, maybe a dict?

Reply:
I will always need a `_workflow.FileList([])` but I guess I can build a dictionary in parallel, use it as a "living documentation" structure and go through it when I need individual `pp_files` elements.

Reply:
Okay, thinking about this more, it is probably better to document this in `pycbc/workflow/grb_utils.py` where `setup_pygrb_pp_workflow` is defined, and here place a reminder to look over there.

Reply:
On second thought (not required for this PR, as it's time to merge this, but something to fix going forward):

I think it's a bad idea to assume anything about the order here, especially things like `[-1]` or `[-2]`. The FileLists were never intended to have a special order. That said, the case of a job that produces multiple output files was never supported well, and dealing with cases like this is ugly ..... Suggestions for improvement here are welcome!

However, I would suggest using one of the methods of the FileList class (https://pycbc.org/pycbc/latest/html/_modules/pycbc/workflow/core.html#FileList) to get these files. It's possible that a new method will be needed, as I think you'll want to search by "description", rather than "tag" (but I'm not sure). I think that `file_list. categorize_by_attr('description')` may produce what @titodalcanton suggests ... and would then be easier and clearer to work with.